kloppy/domain/models/common.py

```diff
@@ -54,6 +65,7 @@ def get_orientation_factor(
         attacking_direction: AttackingDirection,
         ball_owning_team: Team,
         action_executing_team: Team,
+        meta_data: "MetaData",  # TODO is the using before assignment a problem?
```

Comment:
When referring to MetaData when type hinting I got a warning by the linter saying it it was referenced before declared. Set it as a string to type hint but avoid the warning, but not sure what is the proper way to go about this.

Reply:
Ok. Just to confirm, before my changes it was already MetaData in the models and in the models, reader and serializers of the EPTS format. Also, in the Tracab test files it's TracabMetaData. I can change it here and in the other files as well, but just wanted to confirm that's indeed what you want to do.

Reply:
My mistake indeed. I would like to have it Metadata everywhere. @ivd-git did a replace of data_set with dataset and would to have it consistent.

Reply:
ok! I looked at it and there are quite some places and files where it need to be changed, so I would prefer now to make it part of this pull request for now. I could do it later on on this same pull request or as another issue after this one is merged.